scripts/ci/kubernetes/app/templates/airflow.template.yaml

```diff
@@ -42,33 +42,43 @@ spec:
         name: airflow
     spec:
       initContainers:
-      - name: "init"
-        image: {{AIRFLOW_KUBERNETES_IMAGE}}
-        imagePullPolicy: IfNotPresent
-        volumeMounts:
-        - name: airflow-configmap
-          mountPath: /root/airflow/airflow.cfg
-          subPath: airflow.cfg
-        - name: {{INIT_DAGS_VOLUME_NAME}}
-          mountPath: /root/airflow/dags
-        - name: test-volume
-          mountPath: /root/test_volume
-        env:
-        - name: SQL_ALCHEMY_CONN
-          valueFrom:
-            secretKeyRef:
-              name: airflow-secrets
-              key: sql_alchemy_conn
-        command:
-          - "bash"
-        args:
-          - "-cx"
-          - "/tmp/airflow-test-env-init.sh"
+        - name: volume-permissions
+          image: busybox
+          command: ['sh', '-c', 'chmod -R og+rwX /home/airflow/airflow_home/dags /home/airflow/airflow_home/logs /home/airflow/test_volume/']
+          volumeMounts:
+            - name: {{INIT_DAGS_VOLUME_NAME}}
+              mountPath: /home/airflow/airflow_home/dags
+            - name: airflow-logs
+              mountPath: /home/airflow/airflow_home/logs
+            - name: test-volume
+              mountPath: /home/airflow/test_volume
+        - name: "init"
+          image: {{AIRFLOW_KUBERNETES_IMAGE}}
+          imagePullPolicy: IfNotPresent
+          volumeMounts:
+          - name: airflow-configmap
+            mountPath: /home/airflow/airflow_home/airflow.cfg
+            subPath: airflow.cfg
+          - name: {{INIT_DAGS_VOLUME_NAME}}
+            mountPath: /home/airflow/airflow_home/dags
+          - name: test-volume
+            mountPath: /home/airflow/test_volume
+          env:
+          - name: SQL_ALCHEMY_CONN
+            valueFrom:
+              secretKeyRef:
+                name: airflow-secrets
+                key: sql_alchemy_conn
+          command:
+            - "bash"
+          args:
+            - "-cx"
+            - "/tmp/airflow-test-env-init.sh"
 {{INIT_GIT_SYNC}}
       containers:
       - name: webserver
         image: {{AIRFLOW_KUBERNETES_IMAGE}}
-        imagePullPolicy: IfNotPresent
+        imagePullPolicy: Never
```

Comment:
Any reason to change this to `Never`?

Reply:
We deploy the image to cluster via "kind load"